src/main/java/com.gigssandbox/UserCommandHandler.java

```diff
@@ -114,15 +126,15 @@ private Result joinGig(String headliner, String gigDateText) {
         }  catch (IncorrectDateValueException e) {
             return Result.INCORRECT_DATE_VALUE;
 
-        }  catch (NoAppropriateGigsException e) {
-            return Result.NO_APPROPRIATE_GIGS;
+        }  catch (NoAppropriateGigException e) {
```

Comment:
there are two spaces between "}" and "catch". remove one. same for others.

You can create **separate** commit with CTRL+ALT+SHIFT+L IDEA refactor for all source code.

Reply:
this shortcut does not work at my notebook